app/resources/css-default.template

```diff
@@ -607,6 +612,16 @@ td.submit {
   line-height: 10px;
 }
 
+@media (min-width: 700px) {
+  .main .action-wrapper {
+    display: inline-block;
+  }
+  .main .action-outer {
+    float: left;
```

Comment:
Use `float: {{start}}` instead, which flips left/right on RTL.

Reply:
Thanks! It turns out you were right about inline-block (below) so I don't need to use float after all.